pkg/crdpuller/discovery.go

```diff
@@ -92,133 +107,161 @@ func (sp *schemaPuller) PullCRDs(context context.Context, resourceNames ...strin
 	for _, apiResourcesList := range apiResourcesLists {
 		gv, err := schema.ParseGroupVersion(apiResourcesList.GroupVersion)
 		if err != nil {
-			klog.Errorf("skipping discovery due to error parsing GroupVersion %s: %v", apiResourcesList.GroupVersion, err)
+			klog.Errorf("CRDPuller: Skipping discovery due to error parsing GroupVersion %s: %v", apiResourcesList.GroupVersion, err)
```

Comment:
If klog is used here mainly to get line numbers, I think we could remove the CRDPuller prefix, wdyt?